src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -236,9 +248,6 @@ export class SmartSnippet extends Component {
     if (this.lastRenderedResult) {
       this.renderSource();
     }
-    await this.shadowLoading;
-    await Utils.resolveAfter(0); // `scrollHeight` isn't instantly detected, or at-least not on IE11.
-    this.heightLimiter.onContentHeightChanged();
```

Comment:
One of the main improvements with the changes to `AttachShadowPolyfill` is that this is no longer necessary. I tested with IE11, and using the above `handleAnswerSizeChanged` works perfectly.